extensions/amp-inline-gallery/1.0/README.md

```diff
@@ -0,0 +1,237 @@
+---
+$category@: layout
+formats:
+  - websites
+teaser:
+  text: Displays multiple similar pieces of content along a horizontal axis, with optional pagination dots and thumbnails.
+experimental: true
+bento: true
+---
+
+# Bento Inline Gallery
+
+## Usage
+
+The Bento Inline Gallery component uses a Bento Carousel component to display slides, with optional pagination dots and thumbnails. Both components must be properly installed for the environment (Web Component vs Preact). Typical usage might look like:
+
+[example preview="inline" playground="true" imports="bento-inline-gallery:1.0,bento-base-carousel:1.0"]
+
+```html
+<bento-inline-gallery>
+  <bento-inline-gallery-thumbnails aspect-ratio="1.5" loop></bento-inline-gallery-thumbnails>
+  <bento-base-carousel snap-align="center" visible-count="1.2" loop>
+    <img src="server.com/static/inline-examples/images/image1.jpg">
+    <img src="server.com/static/inline-examples/images/image2.jpg">
+    <img src="server.com/static/inline-examples/images/image3.jpg">
+  </bento-base-carousel>
+  <bento-inline-gallery-pagination inset></bento-inline-gallery-pagination>
+</bento-inline-gallery>
+```
+
+[/example]
```

Comment:
This section can be removed since a better example is provided in the ### Web component section

Reply:
removed